src/main/java/org/ow2/proactive/procci/model/occi/metamodel/constants/Attributes.java

```diff
@@ -57,7 +57,7 @@
                     "with the Mixin instance.").build();
     public static final Attribute MIXINS = new Attribute.Builder(MIXINS_NAME).type(Type.LIST).mutable(
             false).required(false).description("Mixin instances associated to this en" +
-            "tity instance. Consumers can expect the BigDataAttributes and Actions of the as" +
+            "tity instance. Consumers can expect the Attributes and Actions of the as" +
```

Comment:
is it normal that some words are splitted (code readibility)?  Other occurencies..